junit-test/output-jvm/src/test/scala/org/scalajs/junit/utils/JUnitTestPlatformImpl.scala

```diff
@@ -37,7 +37,7 @@ object JUnitTestPlatformImpl {
   }
 
   def writeLines(lines: List[String], file: String): Unit =
-    Files.write(Paths.get(file), lines.toIterable.asJava, UTF_8)
+    Files.write(Paths.get(file), (lines: Iterable[String]).asJava, UTF_8)
```

Comment:
out of curiosity, why does this need to be converted specifically to the less specific `JavaCollectionWrappers.IterableWrapper` rather than `JavaCollectionWrappers.SeqWrapper`?

Reply:
`SeqWrapper` seems to be index-based:
https://github.com/scala/scala/blob/207c01b8efa96b9ba556bedcffc13458be2eafbd/src/library/scala/collection/convert/JavaCollectionWrappers.scala#L81
which is not good for a linear collection like `List`. Explicitly ascribing to `Iterable` makes it 100% sure that it is going to use a linear implementation.

Reply:
that actually stopped working in 2.13, as `toIterable` on subtypes of `Iterable[_]` was changed to return `this.type`